CONTRIBUTING.md

```diff
@@ -195,6 +195,19 @@ Check the [OS dependencies](https://superset.incubator.apache.org/installation.h
     superset runserver -d
 
 
+### Logging to the browser console
+
+When debugging your application, you can have the server logs sent directly to the browser console:
+
+    superset runserver -d --console-log
+
+You can log anything to the browser console, including objects:
+
+    from superset import app
+    app.logger.error('An exception occurred!')
+    app.logger.info(form_data)
```

Comment:
oh does `logging.debug('Foo')` work?

Reply:
Let me give it a try.

Reply:
@mistercrunch, currently it won't work because `console_log` adds the websocket handler to a single logger instance ­— Flask uses it's own (`flask.app`), and it's different from the root logger. I'll modify the library so that it takes more than one logger instance, should be straightforward.